mchf-eclipse/drivers/ui/menu/ui_menu.c

```diff
@@ -554,6 +558,26 @@ const char* UiMenu_GetSystemInfo(uint32_t* m_clr_ptr, int info_item)
     case INFO_FLASH:
             snprintf(out,32,"%d",(STM32_GetFlashSize()));
             break;
+//[QBS]s
+	case INFO_UUID_P1:
+	// do something with the first 32bits of unique 96 bits
```

Comment:
If you would use ```snprintf(out,32," 0x%08X%08X%08X\n", STM32_UUID[0],STM32_UUID[1],STM32_UUID[2]);```
you don't need 3 menu entries to do the job and we shoud have enough characters (about 33 in total AFAIR) . And in any case please move this menu item down to the end of the list. The UUID is a very infrequently required information, as it does not bear any technical relevance except ownership maybe as it is just that, a unique 96 bit number different for each STM32 MCU with al least based on my knowledge no additional relevant meaning for us.